CUJExecutor/app/src/androidTest/java/com/example/firstapp/ShellUtility.java

```diff
@@ -547,10 +547,10 @@ public void logData(long[][] allActionStamps) {
      * @param iterations The number of times to run through and measure the CUJ
      * @param recordIntent Whether the user intends to record the test
      ******************************************************************************/
-    public void executeCUJ(String preStr, String postStr, int iterations, boolean recordIntent) throws Exception {
+    public void testCuj(String preStr, String cujStr, int iterations, boolean recordIntent) throws Exception {
```

Comment:
I'd say `testCuj` is a bad name, because it is not a junit test and it isn't testing anything.